api/qsharp/microsoft.quantum.logical.xor.yml

```diff
@@ -0,0 +1,25 @@
+### YamlMime:QSharpType
+uid: microsoft.quantum.logical.xor
+name: Xor
+type: function
+namespace: Microsoft.Quantum.Logical
+summary: Returns the Boolean exclusive disjunction of two values.
+syntax: 'function Xor (a : Bool, b : Bool) : Bool'
```

Comment:
This one opposed to all other docs does not seem to have a remark with the ^^^ syntax.How come?

Reply:
I thought `^^^` was bitwise, not Boolean?

Reply:
Ah, sorry, yes, did not read carefully enough...